hack/boilerplate.sh

```diff
@@ -18,7 +18,7 @@ set -e
 DIR=$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )
 BOILERPLATEDIR=$DIR/boilerplate
 
-files=$(python ${BOILERPLATEDIR}/boilerplate.py --rootdir . --boilerplate-dir ${BOILERPLATEDIR})
+files=$(python3 ${BOILERPLATEDIR}/boilerplate.py --rootdir . --boilerplate-dir ${BOILERPLATEDIR})
```

Comment:
nit: might make sense to split out as this is already a large PR.  Ok to keep though